docs/components/ProgressBar/Mobile.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react-native";
+import { ProgressBar } from "@jobber/components-native";
+
+export default {
+  title: "Components/Status and Feedback/ProgressBar/Mobile",
+  component: ProgressBar,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+    viewport: { defaultViewport: "mobile1" },
+  },
+} as ComponentMeta<typeof ProgressBar>;
+
+const BasicTemplate: ComponentStory<typeof ProgressBar> = args => (
+  <ProgressBar {...args} />
+);
+
+export const Basic = BasicTemplate.bind({});
```

Comment:
Do you think it would be beneficial to add an example that uses the `header` prop?

Reply:
Good idea @MichaelParadis!  Added, matching the web state example. Let me know what you think.